continuous_integration/travis/install.sh

```diff
@@ -61,14 +61,7 @@ install_conda()
 install_pip()
 {
     # Install PyQt5
-    conda install 'pyqt>=5.6.0';
-
-    # Install testing packages
-    pip install pytest pytest-cov pytest-qt mock
-
-    # Install extra packages
-    EXTRA_PACKAGES="matplotlib pandas sympy pyzmq pillow"
-    pip install $EXTRA_PACKAGES
```

Comment:
Please revert this change for now. I have to plan on how to start using the pyqt5 wheels on CircleCI.